Symfony/CS/Tests/Fixer/Symfony/PhpdocTypesFixerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+class PhpdocTypesFixerTest extends AbstractFixerTestBase
```

Comment:
Classes that were already existing on 1.x line cannot become finale to not break BC, but new files can be marked as final.
Also, internal could be good here ;)